lib/exec.js

```diff
@@ -401,6 +401,7 @@ function executeFirejail(command, args, options) {
     } else {
         baseOptions.push('--private');
     }
+    baseOptions.push(`--env=TMPDIR=/tmp`);
```

Comment:
Interesting this was needed, and great fine! :) Seems safe (CC @apmorton in case his clever mind can see this as a potential issue).

Reply:
Right, this is complicated. If you just do make on tendra, it resolves to /tmp naturally, but if you supply a prefix to install to it will default to /install/folder/tmp. So I added to the Tendra sourcecode that if TMPDIR is set, it uses that instead of /install/folder/tmp, but we have to pass this on to the compiler, and firejail uses custom env params so it didn't help to add it to tendra.js

Reply:
> if you supply a prefix to install to it will default to /install/folder/tmp

I thought this should only happen during a build, not when installed to `$PREFIX`. Remember the build has several stages; bootstrap and rebuild, and I tried to keep all files self-contained within there during a build.

If we're not using /tmp _after_ installing to `$PREFIX`, let's fix that!